server/src/resolvers/models.ts

```diff
@@ -5,6 +5,22 @@ export const models = [
       publishCreate: true,
       publishUpdate: true,
       publishDelete: true
+    },
+    // this auth config piece should be generated by the graphback plugin
+    // we can discuss whether that's directives or annotations
+    auth: {
+      rules: [
+        {
+          allow: 'owner',
+          operations: ['update', 'create', 'read', 'delete'],
+          ownerField: 'owner',
+        },
+        {
+          allow: 'role',
+          operations: ['update', 'delete'],
+          roles: ["admin"]
+        }
+      ]
```

Comment:
IMO this should be the output of the auth plugin. The plugin would take either directives or annotations on the model and convert it into this config here.